MultiMystery.py

```diff
@@ -180,14 +183,20 @@ def get_alive_threads():
                             tqdm.write("YAML error")
                             break
                         done = check_if_done()
-                        if done:
+                        if done and not keep_all_seeds:
                             break
                     else:
                         msg = f"Seed Attempt #{task.task_id:4} was successful."
 
                         dead_or_alive[task.task_id] = True
+                        alive[task.task_id] = True
                         done = check_if_done()
-                        if done:
+                        if keep_all_seeds:
+                            tqdm.write(msg)
+                            os.makedirs(os.path.join(output_path, basedir, str(task.task_id)))
```

Comment:
you should set exist_ok=True or this will crash if you happen to roll the same seed twice.

Reply:
exist_ok=True added, in the copy_seed version of the function.  copy_seed called directly here now.